app/views/documents/_whitehall_imported_entry.html.erb

```diff
@@ -0,0 +1,10 @@
+<h4 class="govuk-heading-s govuk-!-margin-bottom-1">
+  <%= t "documents.history.entry_types.whitehall_migration.#{entry.details.entry_type}" %>
+</h4>
+
+<div class="app-timeline-entry__dateline">
+  <%= format_date_at_time(entry.created_at) %>
+  <% if entry.created_by %>
+    by <%= name_or_fallback(entry.created_by) %>
```

Comment:
Do we need a fallback if we're checking if the user exists?

Reply:
I think convention would dictate that we'd want the by in a translation file - sorry I know that's a bit of a pain for 2 letters.

Reply:
There should always be a creator given [this](https://github.com/alphagov/content-publisher/blob/master/app/controllers/documents_controller.rb#L28). I guess it just won't display anything if there was no value for `entry.created_by`, seems ok to me but I suppose we could display some text instead?

Reply:
Will change the `by` to be localised